third_party/move/move-compiler-v2/src/pipeline/reference_safety_processor.rs

```diff
@@ -73,13 +74,15 @@ use std::{
 /// `MemoryLocation`, as well as the children of the node, given by list of `BorrowEdge`s.
 /// The node also has backlinks to its parents, given by a set of `LifetimeLabel`, for
 /// more flexible navigation through the (acyclic) graph.
-#[derive(Clone, Debug, PartialEq, Eq)]
+#[derive(AbstractDomain, Clone, Debug, PartialEq, Eq)]
 struct LifetimeNode {
     /// Memory location associated with this node.
```

Comment:
Would be good to document why a particular field is not joined (and why that is okay).

Reply:
+1 but we should let the author to document it.

Reply:
Yeah, this is the authors problem. In the new borrow checker this is resolved.